.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,20 @@
+_Replace this line with a description of your changes. Please follow the suggestions in the comment below._
+
+-----------------
+
+If this PR requires any particular action or consideration before deployment,
+please check the reasons or add your own to the list:
+
+* [ ] Requires a schema migration that can cause downtime. Postpone deployment until maintenance window[1].
```

Comment:
I would use `database migration` or `db migration`. To me `schema migration` sounds a bit generic :)

Reply:
good point, yes